spec/services/discovery_engine/evaluation/evaluation_resource_spec.rb

```diff
@@ -0,0 +1,92 @@
+RSpec.describe DiscoveryEngine::Evaluation::EvaluationResource do
+  let(:sample_set_id) { "clickstream_01_02" }
+  let(:evaluation_resource) { described_class.new(sample_set_id) }
+
+  describe "#fetch_and_output_metrics" do
+    let(:quality_metrics_object) { double("quality_metrics", to_h: "some output") }
+    let(:evaluation_service_stub) { double("evaluation_service", create_evaluation: operation_object) }
+    let(:operation_object) { double("operation", error?: false, wait_until_done!: true, results: response_object) }
+    let(:response_object) { double("response", name: "/evaluations/1") }
```

Comment:
I'd be tempted to leave `_stub` and `_object` off these to keep them short and consistent